packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -91,6 +91,13 @@ type DefaultProps = Readonly<{
      * Whether to display shortcuts for Select All and Select None.
      */
     shortcuts: boolean;
+    /**
+     * When false, the SelectOpener can show a Node as a label. When true, the
+     * SelectOpener will use a string as a label. If using custom OptionItems, a
+     * plain text label can be provided with the `labelAsText` prop.
+     * Defaults to true.
+     */
+    showLabelAsText: boolean;
```

Comment:
super minor: What do you think about naming the prop `showOpenerLabelAsText` so it helps communicate that it's used for the opener? (same for SingleSelect!)